msal/tests/Test.MSAL.NET.Unit/RequestsTests/InteractiveRequestTests.cs

```diff
@@ -325,7 +326,7 @@ public void DuplicateQueryParameterErrorTest()
 
             try
             {
-                request.PreTokenRequestAsync().Wait();
+                request.PreTokenRequestAsync(CancellationToken.None).Wait();
```

Comment:
>CancellationToken.None [](start = 45, length = 22)

these are defaults, right? Shouldn't the API keep working as is if there are no changes made to the client code?

Reply:
Yes, these are the defaults.  You just can't make CancellationToken.None a default parameter to a function since it's not a compile time constant.